mk/tests.mk

```diff
@@ -0,0 +1,17 @@
+.PHONY: test run-test-map
+
+TEST_ARGS_FILE = tests/test-args/test-args.elf
```

Comment:
Rename to DUT ([device under test](https://en.wikipedia.org/wiki/Device_under_test)).

Reply:
i.e., `dut.elf`